src/test.ts

```diff
@@ -14,7 +14,8 @@ describe("redux-reducer-effects", function() {
         it("keeps all state changes", function() {
             const combined = composeReducers(increment, increment, increment);
 
-            const [state] = <any>(combined({ counter: 0 }, { type: "init" }));
+            const result = combined({ counter: 0 }, { type: "init" });
+            const state = getState(result);
```

Comment:
We changed the reducer result type from `[State, Tasks]` to `State | [State, Tasks]` to reduce boilerplate, but worth noting the boilerplate the overhead this adds elsewhere.